src/TablePaginationSort/TablePaginacion.jsx

```diff
@@ -198,7 +198,7 @@ const TablePaginacion = () => {
         pageIndex={pageIndex}
         setPageIndex={setPageIndex}
         totalItemsCount={data.length}
-        pageSizeOptions={[5, 25, 50]}
+        pageSizeOptions={[10, 25, 50]}
```

Comment:
In this part the option is different to the initial state so, the table render 5 items and the selected option is 10 items.

Reply:
You're right! It escaped on my last PR, in this case I made a new commit setting the initial state to 10!